docker/Dockerfile

```
@@ -0,0 +1,31 @@
+## First stage: build the application
+FROM golang:1.13.0-alpine AS builder
+ARG GOARCH
+ARG GOARM
+
+WORKDIR /go/src/github.com/CESARBR/knot-babeltower
+COPY . .
+
+# Install glide dependency manager
+RUN apk update \
+    && apk add make git curl \
+    && curl https://glide.sh/get | sh
+
+# Install project development tools and dependencies
+RUN make tools
+RUN make deps
+
+# Build the application
+RUN make bin
+
+# Second stage: create the entrypoint to the application binary generated in the previous stage
+FROM alpine:latest
```

Is using the `latest` tag a good idea? Personally, I find it quite misleading (_take a look [here](https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375) and [here](https://vsupalov.com/docker-latest-tag/), for more details_). Maybe, it'd be a good idea to specify a version (such as `alpine:3.10`). [Kubernetes](https://kubernetes.io/docs/concepts/configuration/overview/#using-labels) itself doesn't recommend using the `lastest` tag, for the following reasons :eyes: :
> Note: You should avoid using the :latest tag when deploying containers in production as it is harder to track which version of the image is running and more difficult to roll back properly.

```suggestion
FROM alpine:3.10
```

---

No, it's not a good idea. The codacy has warned me and I'll fix.

---

Done. Thanks!